eventtracking/django/models.py

```diff
@@ -142,6 +145,19 @@ def compiled_expressions(self):
         logger.info('Added compiled expressions in cache for filter "%s"', self)
         return compiled_expressions
 
+    @classmethod
+    def generate_cache_key(cls, **kwargs):
```

Comment:
@danialmalik we should move this into common utils instead of model.

Reply:
@ziafazal making this a utility method will not avoid duplication as the params will be the same. I've added the `namespace` parameter in the code where `get_cache_key` is being called.

Reply:
@danialmalik we can create a utility method like this
```
def generate_cache_key(**kwargs):
      kwargs['namespace'] = 'eventtracking.regex_filter'
      return get_cache_key(**kwargs)
```

Reply:
but this way this utility method is only usable with the `regex_filter` caching.